datafusion/src/optimizer/constant_folding.rs

```diff
@@ -107,7 +107,7 @@ impl OptimizerRule for ConstantFolding {
 
                 utils::from_plan(plan, &expr, &new_inputs)
             }
-            LogicalPlan::TableScan { .. } | LogicalPlan::EmptyRelation { .. } => {
+            LogicalPlan::TableScan { .. } | LogicalPlan::EmptyRelation(_) => {
```

Comment:
nit: keep consistent.   `{ .. }` `(_)`

Reply:
perhaps we can do this in a follow on pr. Good spot @xudong963

Reply:
@xudong963 @alamb 
I might miss this point when write the code.
After all the extracting logical plan, i will file a followup pull request to keep consistent.

Reply:
(I merged the PR in before you had a chance to respond to this comment -- so it was my fault :) )